data/layout/telescope_positions-North-1SST.ecsv

```diff
@@ -51,5 +46,5 @@
 #       unit: *id001
 #       value: 3.0
 # schema: astropy-2.0
-telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
-SST-01 0.0 0.0 0.0 28.762158000000007 -17.891571 217654.06411838497 3185065.2899643113 2154.75
+telescope_name      pos_x      pos_y      pos_z
+SST-01               0.0        0.0        0.0
```

Comment:
Is there a reason why we need a 1 SST array for the North?

Reply:
As far as I understand all these files are for the trigger application. I haven't changed them, only the spacing. 

By the way, by testing the `applications/sim_showers_for_trigger_rates` I realized that the log files are being produced but corsika was failing due to the following error:
```
EPSG 32628
 DATAC : UNKNOWN KEYWORD: EPSG
```

You can test it yourself by running:
```
python applications/sim_showers_for_trigger_rates.py --array 4LST --site North --primary proton --nruns 2 --nevents 10 --submit_command local
```
and checking the logfiles, which locations are printed out in the terminal. Should I open an issue for that?

Reply:
This is because EPSG was added to the `corsika_parameters.yml` file and as a result is written out to the CORSIKA input file. The fix should be simply removing that parameter (we read it from the DB now anyway), but I will open an issue just in case it is a bit more complex.

Reply:
Oh and regarding Gernot's original point, this file should be removed all together, we don't need an SST in the north. If we don't have an SST in the south, we can modify this file to be for the south instead.

Reply:
I understand. I am then removing the EPSG from the file. Wouldn't this be the case where we cannot be sure about the future (who knows about the next CTA upgrades)? We could keep this file if you don't mind.